### ISSUE TITLE
engineering.da-int.net references

### ISSUE THREAD

**Issue by jonas-chevalier-da** [digital-asset/daml#160]
To make daml usable by the public, all the internal engineering.da-int.net need to be replaced with public-accessible equivalents. See #304 for the epic.

```
dev-env/bin/dade-copyright-headers
17:# http://engineering.da-int.net/docs/engineering-handbook/licenses-copyrights.html

dev-env/bin/da-sdk-head
20:  echo "Please visit https://engineering.da-int.net/sdk/summary/releases/latest/doc/packages/sdk-docs-installation/"

dev-env/windows/manifests/java-oraclejdk-8u111.json
8:      "url": "https://engineering.da-int.net/nix-vendored/jdk-8u111-windows-x64.exe#/dl.7z",
12:      "url": "https://engineering.da-int.net/nix-vendored/jdk-8u111-windows-i586.exe#/dl.7z",
40:      "dl https://engineering.da-int.net/nix-vendored/jce_policy-8.zip \"$dir\\tmp\\jce_policy-8.zip\"",

dev-env/windows/manifests/README.md
40:In general, binaries are provided from: https://engineering.da-int.net/nix-vendored/<tool_name>/<file_name>

dev-env/windows/manifests/vcredist-14.0.23026.json
9:    "url": "https://engineering.da-int.net/nix-vendored/vc_redist/vc_redist.x64.14.0.23026.exe",

dev-env/windows/README.md
11:    iex (new-object net.webclient).downloadstring('https://engineering.da-int.net/download/dadew-installer')

dev-env/windows/dadew-installer.ps1
38:(new-object net.webclient).downloadFile("https://engineering.da-int.net/download/dadew.zip", $zipFile)

ledger/sandbox/README.md
41:The dar files are the archives containing compiled DAML code. We highly recommend generating the dar files using the new DAML packaging, as described in https://engineering.da-int.net/docs/da-all-docs/packages/daml-project/. This will ensure that you're generating the .dar files correctly. The linked page also gives a good overview of what the dar files are, along with other key concepts.
```

**Comment by associahedron** [comment]
They're all gone.

**Comment by associahedron** [comment]
Wait nevermind, there's still a number of them. I fail at grep. 

Biggest offenders is `dev-env`. Second biggest is `ledger/sandbox/README.md`.

**Comment by garyverhaegen-da** [comment]
@associahedron here you go: #2485.

**Comment by garyverhaegen-da** [comment]
Fixed by #2485.
